src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK = 'CHECK';
+
```

Comment:
- [ ] Same here, Please edit this action type to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).